src/ml/neural_net/mps_layer_instance_norm_data_loader.mm

```diff
@@ -95,15 +95,19 @@ - (instancetype) initWithParams:(NSString *)name
                                                    dataType:(MPSDataTypeFloat32)];
 
     _style_props = [[NSMutableArray alloc] init];
-    
+
     for (NSUInteger index = 0; index < styles; index ++) {
       TCMPSInstanceNormDataLoaderProps *style_property = [[TCMPSInstanceNormDataLoaderProps alloc] init];
 
-      id<MTLBuffer> gammaBuffer = [dev newBufferWithBytes:_gamma_weights.bytes + (index * _numberOfFeatureChannels * sizeof(float))
+      size_t gammaBufferOffset = _gamma_weights.bytes + (index * _numberOfFeatureChannels * sizeof(float));
```

Comment:
These aren't offsets, these are addresses. Just factor out the offset, which is shared: `(index * _numberOfFeatureChannels * sizeof(float)`